src/app/pages/Main/Main.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styles from './Main.module.css';
+import Card from '../../components/Card/Card';
+import Hero from '../../components/Hero/Hero';
+import NavBar from '../../components/NavBar/NavBar';
+
+export default function Main(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <header>
+        <Hero />
+      </header>
+      <main className={styles.cards}>
+        <div className={styles.card}>
```

Comment:
I'm not sure if it is really necessary to put the Card components into a div each. Could the className be given to the Card directly?

I imagine you will map the cards into the main later on.